po/extra/fr.po

```diff
@@ -44,45 +44,97 @@ msgid ""
 msgstr ""
 
 #: data/com.github.ryonakano.reco.appdata.xml.in:16
-#, fuzzy
 msgid ""
 "Saving in some commonly used formats. It supports AAC, FLAC, MP3, Ogg "
 "Vorbis, Opus, and WAV"
-msgstr "Formats disponibles : AAC, FLAC, MP3, Ogg Vorbis, Opus et WAV"
+msgstr ""
+"Enregistrement dans les formats les plus courant. Formats pris en charge : "
+"AAC, FLAC, MP3, Ogg Vorbis, Opus et WAV"
 
 #: data/com.github.ryonakano.reco.appdata.xml.in:17
 msgid ""
 "Timed recording. You can set a delay before recording up to 15 seconds, and "
 "the length of recording up to 600 seconds"
 msgstr ""
+"Enregistrement différé et chronométré. Vous pouvez définir un délai avant "
+"d'enregistrer jusqu'à 15 secondes, et la durée de l'enregistrement jusqu'à "
+"600 secondes"
 
 #: data/com.github.ryonakano.reco.appdata.xml.in:18
 msgid ""
 "Autosaving or always-ask-where-to-save workflow. You can select whether the "
 "app saves recordings automatically or you choose a destination each time on "
 "saving them"
 msgstr ""
+"Comportement de sauvegarde automatique ou demande d'emplacement de "
+"destination pour chaque enregistrement. Vous pouvez sélectionner si "
+"l'application sauvegarde automatiquement les enregistrements ou si vous "
+"préferez sélectionner un emplacement de destination à chaque sauvegarde."
```

Comment:
The translated string seems to have a period which the source string does not have at the end, but is it intentional?

Reply:
It's not intentional, I'll remove it.